afew/MailMover.py

```diff
@@ -52,7 +56,7 @@ def move(self, maildir, rules):
         to_delete_fnames = []
         moved = False
         for query in rules.keys():
-            destination = '{}/{}/cur/'.format(self.db_path, rules[query])
```

Comment:
Where did that `cur` here go?

Reply:
Apparantly, #200 and #201 got merged together when they were both merged (or I rebased one PR branch too early, or github computes the diff relative to the original merge base).
In any case, this is reflected in the change to new_name(): In short, afew used to move mail from new to cur but failed to update flags (everyone moving to cur should update flags), so #201 made afew leave mail in the respective subdir rather than chopping off new/cur/.. and appending cur..

Reply:
Sorry, but this still doesn't feel quite readable to me.

I could guess `submaildir` is `cur` or `new`, but just taking `[1]` out of `os.path.split()`'s `tail` sounds scary - what happens to the other path components?

Could this maybe be made slightly more readable way, maybe by adding some comments?

We could also probably use `pathlib` paths around instead of strings.